ctera/src/main/java/ch/cyberduck/core/ctera/CteraMoveFeature.java

```diff
@@ -23,6 +23,8 @@
 
 import java.text.MessageFormat;
 
+import static ch.cyberduck.core.ctera.CteraAclPermissionFeature.*;
+
 public class CteraMoveFeature extends DAVMoveFeature {
```

Comment:
Add test showing `guid` is retained when moving file.

Reply:
@dkocher this is an integration test, right? (As the guid comes from the portal).

Reply:
Yes, as in `CteraMoveFeatureTest`

Reply:
29234bc512ea0d073697d20e90a992425a3e14db